contracts/exchange-forwarder/test/utils/forwarder_test_factory.ts

```diff
@@ -0,0 +1,359 @@
+import { ERC20Wrapper } from '@0x/contracts-asset-proxy';
```

Comment:
This file is really nice! It cuts down a lot on the logic in the forwarder tests